crypto/evp/m_sigver.c

```diff
@@ -181,6 +181,9 @@ static int do_sigver_init(EVP_MD_CTX *ctx, EVP_PKEY_CTX **pctx,
              * an old digest.
              */
             evp_md_ctx_clear_digest(ctx, 1);
+            /* legacy code support for engines */
+            ERR_set_mark();
```

Comment:
If we don't end up calling `ERR_pop_to_mark()`, then all other code paths should end up calling `ERR_clear_last_mark()`.

Reply:
Done. Thanks!